clock/testing/fake_clock.go

```diff
@@ -198,12 +198,10 @@ func (f *FakeClock) setTimeLocked(t time.Time) {
 			if w.skipIfBlocked {
 				select {
 				case w.destChan <- t:
-					w.fired = true
 				default:
```

Comment:
This field is still defined in the struct, L51

Reply:
oops, sorry about that
removed